zScanner/Services/Storage/Model/PageDatabaseModel.swift

```diff
@@ -14,6 +14,7 @@ class PageDatabaseModel: Object {
     @objc dynamic var index = 0
     @objc dynamic var correlationId = ""
     @objc dynamic var relativePath = ""
+//    @objc dynamic var descript = ""
```

Comment:
don't commit commented code. 
Don't make the name of the variable shorter. Any shortening can bring confusion. we have autocomplete and sometimes it's even working 😅

Reply:
I know but it says that I cannot use "description" because it's already used. Should I use a different name e.g. notes but I must change it everywhere.
It says exactly this "Cannot override with a stored property 'description'", "Getter for 'description' with Objective-C selector 'description' conflicts with getter for 'description' from superclass 'Object' with the same Objective-C selector"

Reply:
Stupid Realm 🙄 Yeah, use notes, or userDesription or something. You don’t have to change it everywhere. That’s what the mapper is for. Just convert DatabaseModel.userDescription into DomainModel.description

Reply:
Done but It gonna need to migrate and I have no idea how to do it.